src/main/java/com/minecolonies/coremod/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -209,6 +215,47 @@ public Class getExpectedBuildingClass()
         return BuildingLumberjack.class;
     }
 
+    /**
+     * The lumberjack is a special worker.
+     * In his/her decision state, he/she will try to add lumberjack cycles
+     * If there's nothing left to craft, he/she will proceed with woodworking
```

Comment:
maybe just use "they" here in the javadoc, or "it" lol

Reply:
Changed in 7541035